packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -193,6 +193,10 @@ export class ProjectService {
         );
 
         const adapter = await ProjectService.testProjectAdapter(updatedProject);
+        const explores = await adapter.compileAllExplores();
```

Comment:
@rephus this is going to timeout and not use the new project compile UI. To trigger that you'd need to kick off a job and return a job id

Then have the frontend set the `active_job` state to trigger the polling